### PR TITLE
disable odb file support for libreoffice

### PR DESCRIPTION
**Pull request by inkhey**
related to #59

### CODE REVIEW

File: preview_generator/preview/builder/office__libreoffice.py [hunk 1]
```diff
@@ -117,8 +117,10 @@ def convert_office_document_to_pdf(
 # Code duplicated from https://raw.githubusercontent.com/LibreOffice/core/master/bin/get-bugzilla-attachments-by-mimetype
 LO_MIMETYPES = {
     # ODF
-    'application/vnd.oasis.opendocument.base': 'odb',
-    'application/vnd.oasis.opendocument.database': 'odb',
+    # INFO - G.M - 2018-11-13 - Convertion of base to pdf is not correctly
```
Comment (buxx) [inline preview_generator/preview/builder/office__libreoffice.py]: Typo "Conversion" instead "Convertion". I change it myself.

**Comment by buxx** [comment]
I will do merge with  squash.

**Comment by inkhey** [comment]
@buxx is it done ?

**Comment by buxx** [comment]
@inkhey Sorry for the late. It's done.

**Pull request merged**